src/cdi_eospac/Eospac.cc

```diff
@@ -453,27 +453,26 @@ void Eospac::expandEosTable() const {
 
   EOS_INTEGER errorCode(0);
   Check(returnTypes.size() < INT32_MAX);
-  int nTables(static_cast<int>(returnTypes.size()));
+  EOS_INTEGER nTables(static_cast<int>(returnTypes.size()));
```

Comment:
@KineticTheory Sorry, one more `static_cast<int>` here

Reply:
Yes.  Those are hard for me to see.  Thanks for pointing it out.